bin/improver-nbhood

```diff
@@ -78,6 +79,10 @@ def main():
                        'should be used. For example: 10000,12000,14000 1,2,3 '
                        'where a lead time of 1 hour uses a radius of 10000m, '
                        'a lead time of 2 hours uses a radius of 12000m, etc.')
+    parser.add_argument('--wind_direction', action='store_true', default=False,
```

Comment:
Maybe this should be `--angle` or `--degrees` rather than referring to a specific diagnostic?